indirect_test.cc

```diff
@@ -325,6 +331,11 @@ TEST(IndirectTest, HashValueless) {
   EXPECT_EQ(std::hash<xyz::indirect<int>>()(v), static_cast<size_t>(-1));
 }
 
+TEST(IndirectTest, InitializerListConstructor) {
+  xyz::indirect<std::vector<int>> i(xyz::in_place_t{}, {1, 2, 3});
```

Comment:
Perhaps add a test for a list of 2, as this was the case which resulted in an error which originally led to the additional constructor being called to avoid calling the vectors(size_t, size_t) constructor inadvertently.